lib/eofs/standard.py

```diff
@@ -152,7 +158,18 @@ def __init__(self, dataset, weights=None, center=True, ddof=1):
         dataNoMissing = self._data[:, nonMissingIndex]
         # Compute the singular value decomposition of the design matrix.
         try:
-            A, Lh, E = np.linalg.svd(dataNoMissing, full_matrices=False)
+            if has_dask and isinstance(dataNoMissing, dask.array.Array):
+                # Use the parallel Dask algorithm
+                dsvd = dask.array.linalg.svd(dataNoMissing)
+                A, Lh, E = (x.compute() for x in dsvd)
+            else:
+                # Basic numpy algorithm
+                A, Lh, E = np.linalg.svd(dataNoMissing)
+
+            # Trim the arrays (since Dask doesn't support
+            # 'full_matrices=False')
+            A = A[:, :len(Lh)]
+            E = E[:len(Lh), :]
```

Comment:
Just a comment for discussion. Dask also has a [`svd_compressed`](http://docs.dask.org/en/latest/array-api.html#dask.array.linalg.svd_compressed) function.

> This computes the approximate singular value decomposition of a large array. This algorithm is generally faster than the normal algorithm but does not provide exact results. One can balance between performance and accuracy with input parameters (see below).

You have to explicitly specify the rank of the desired thin SVD decomposition.

Obviously this is not part of the numpy API. Nevertheless, it seems like it would could be useful to have the option to use it when dealing with dask arrays (which are presumably very large).